src/TicTacToe.hs

```diff
@@ -87,25 +87,25 @@ findAndReplace :: Board -> Piece -> Piece -> Board
 findAndReplace b p1 p2 = listToBoard [if x==p1 then p2 else x | x <- bl]
 	where bl = boardToList b
 
--- |Check if X's can immediately win, and if so, do it
+-- |Check if O's can immediately win, and if so, do it
 winningXMove :: Board -> Maybe Board
-winningXMove b = headMay [findAndReplace b p (Right X) | p <- (possibleMoves b), won (findAndReplace b p (Right X))]
+winningXMove b = headMay [findAndReplace b p (Right O) | p <- (possibleMoves b), won (findAndReplace b p (Right O))]
```

Comment:
@2016rshah Sorry, my bad, I thought I had changed them!

Will do it shortly and push the changes up :)